lib/node_modules/@stdlib/math/base/special/expm1rel/README.md

```diff
@@ -87,7 +96,6 @@ var v = expm1rel( 0.0 );
 ```javascript
 var expm1rel = require( '@stdlib/math/base/special/expm1rel' );
```

Comment:
The code from the `example/index.js` file should be included here with the exception that 

```javascript
var expm1rel = require( './../lib' );
```
in the example file be the package name when included in the README

```javascript
var expm1rel = require( '@stdlib/math/base/special/expm1rel' );
```